src/shogun/classifier/AveragedPerceptron.h

```diff
@@ -52,37 +47,33 @@ class CAveragedPerceptron : public CLinearMachine
 		/// set learn rate of gradient descent training algorithm
 		inline void set_learn_rate(float64_t r)
 		{
-			learn_rate=r;
+			learn_rate = r;
 		}
 
 		/// set maximum number of iterations
 		inline void set_max_iter(int32_t i)
```

Comment:
i dont think we need this setter here @karlnapf  ?

Reply:
nope, but we also don't need to delete those (yet). But for sure we dont need to add new ones

Reply:
we can make this new setter typed